top.nix

```diff
@@ -0,0 +1 @@
+import <nixpkgs> { localSystem = { config = "x86_64-unknown-linux-musl"; }; }
```

Comment:
and similarly, accept a nixpkgs path as an input, so it doesnt rely on the current value of `$NIX_PATH`